src/app/pages/contact/embedded.scss

```diff
@@ -0,0 +1,30 @@
+@import 'pattern-library/core/pattern-library/headers';
+
+.embedded-contact {
```

Comment:
We probably need some padding to push the text over or down to accommodate the close button:
![image](https://github.com/user-attachments/assets/253e0b0a-7aba-4923-af9d-5521395cfbec)

(and I think this is on your assignments PR, but seems like some padding issue with that button?)

Reply:
I have reworked the put-away as a back button in a footer (like the design).
PR for that is:
https://github.com/openstax/assignments/pull/192

Reply:
It might need some `position: fixed` rules? But I'll let ya'll work that out on that PR 
![image](https://github.com/user-attachments/assets/f54c2bd6-ead3-44a7-8568-fb14be9d93c7)